### PR TITLE
What's New: Correlations in 10.2

### PR DESCRIPTION
**Pull request by ifrost**
Update on what's new in Correlations in 10.2

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 1]
```diff
@@ -67,3 +67,23 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
...
+
+## Correlations Editor in Explore
+
+<!-- Kristina Durivage -->
+
+_Available in public preview in Grafana_
+
+Creating correlations has just become easier. Check out our new correlations editor inside Explore by selecting the `+ Add > Add correlation` option from the top bar, or from the command palette. The editor shows all possible places where data links can be placed. The editor will guide you through building and testing target queries. For more details please check [the documentation]({{< relref "../administration/correlations/" >}}).
+
+To try out Correlations, enable the `correlations` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, please contact customer support.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:79]:
```suggestion
To try out Correlations, enable the `correlations` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, contact Grafana Support.
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 2]
```diff
@@ -67,3 +67,23 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
...
+
+## Create correlations for provisioned data sources
+
+<!-- Piotr Jamróz -->
+
+_Available in public preview in Grafana_
+
+You can now create correlations either using administration page or provisioning regardless of whether data source was provisioned or not. In previous versions there was a restriction that if a data source was provisioned, the only way to add correlations to it was also with provisioning. It's no longer the case and you can easily create new correlations mixing both methods - using administration page and/or provisioning.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:87]:
```suggestion
You can now create correlations using either the **Administration** page or provisioning, regardless of whether a data source was provisioned or not. In previous versions of Grafana, if a data source was provisioned, the only way to add correlations to it was also with provisioning. Now, that's no longer the case, and you can easily create new correlations mixing both methods—using the **Administration** page or provisioning.
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 3]
```diff
@@ -67,3 +67,23 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
...
+
+To try out Correlations, enable the `correlations` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, please contact customer support.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:89]:
```suggestion
To try out Correlations, enable the `correlations` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, contact Grafana Support.
```

**Review by imatwawana — COMMENTED** [review]
Thanks for contributing. I've made some wording and style edits and left some questions about availability. Once we have this version ready to go, I'll add these items to the Cloud release notes as well.

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 4]
```diff
@@ -67,3 +67,23 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
...
+
+<!-- Kristina Durivage -->
+
+_Available in public preview in Grafana_
+
+Creating correlations has just become easier. Check out our new correlations editor inside Explore by selecting the `+ Add > Add correlation` option from the top bar, or from the command palette. The editor shows all possible places where data links can be placed. The editor will guide you through building and testing target queries. For more details please check [the documentation]({{< relref "../administration/correlations/" >}}).
+
+To try out Correlations, enable the `correlations` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, please contact customer support.
+
+## Create correlations for provisioned data sources
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:81]:
```suggestion
### Create correlations for provisioned data sources
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 5]
```diff
@@ -67,3 +67,23 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
...
 
 With the current release, we enabled RBAC permission validation (`rbac.permission_validation_enabled` setting) by default. This means that the permissions provided in the request during custom role creation or update are validated against the list of [available permissions and their scopes](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/custom-role-actions-scopes/#action-definitions). If the request contains a permission that is not available or the scope of the permission is not valid, the request is rejected with an error message.
+
+## Correlations Editor in Explore
+
+<!-- Kristina Durivage -->
+
+_Available in public preview in Grafana_
+
+Creating correlations has just become easier. Check out our new correlations editor inside Explore by selecting the `+ Add > Add correlation` option from the top bar, or from the command palette. The editor shows all possible places where data links can be placed. The editor will guide you through building and testing target queries. For more details please check [the documentation]({{< relref "../administration/correlations/" >}}).
```
[the remark this answers is not in the document]
Comment (ifrost) [reply docs/sources/whatsnew/whats-new-in-v10-2.md:79]: Sweet! Thanks ❤️  Might be worth updating the template as well, currently it says:

> You must use relative references when linking to docs within the Grafana repo. Please do not use absolute URLs.

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 6]
```diff
@@ -67,3 +67,23 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
 _Generally available in Grafana Enterprise_
 
 With the current release, we enabled RBAC permission validation (`rbac.permission_validation_enabled` setting) by default. This means that the permissions provided in the request during custom role creation or update are validated against the list of [available permissions and their scopes](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/custom-role-actions-scopes/#action-definitions). If the request contains a permission that is not available or the scope of the permission is not valid, the request is rejected with an error message.
+
+## Correlations Editor in Explore
+
+<!-- Kristina Durivage -->
+
+_Available in public preview in Grafana_
```
[the remark this answers is not in the document]
Comment (ifrost) [reply docs/sources/whatsnew/whats-new-in-v10-2.md]: In both cases it's OSS, Enterprise and Cloud. Updated the description, though I'm not sure if we need to explicitly mention Cloud 🤔

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 7]
```diff
@@ -67,3 +67,23 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
...
+
+Creating correlations has just become easier. Check out our new correlations editor inside Explore by selecting the `+ Add > Add correlation` option from the top bar, or from the command palette. The editor shows all possible places where data links can be placed. The editor will guide you through building and testing target queries. For more details please check [the documentation]({{< relref "../administration/correlations/" >}}).
+
+To try out Correlations, enable the `correlations` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, please contact customer support.
+
+## Create correlations for provisioned data sources
+
+<!-- Piotr Jamróz -->
+
+_Available in public preview in Grafana_
```
[the remark this answers is not in the document]
Comment (ifrost) [reply docs/sources/whatsnew/whats-new-in-v10-2.md]: Same as above: 

> In both cases it's OSS, Enterprise and Cloud. Updated the description, though I'm not sure if we need to explicitly mention Cloud 🤔

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 8]
```diff
@@ -67,3 +67,25 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
...
 
 With the current release, we enabled RBAC permission validation (`rbac.permission_validation_enabled` setting) by default. This means that the permissions provided in the request during custom role creation or update are validated against the list of [available permissions and their scopes](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/custom-role-actions-scopes/#action-definitions). If the request contains a permission that is not available or the scope of the permission is not valid, the request is rejected with an error message.
+
+## Correlations
+
+### Correlations editor in Explore
+
+<!-- Kristina Durivage -->
+
+_Available in public preview in Grafana Open Source, Enterprise and Cloud_
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:77]: Removing mention of the Cloud availability here and I'll add these to the Cloud What's new and ping you for review
```suggestion
_Available in public preview in Grafana Open Source and Enterprise_
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 9]
```diff
@@ -67,3 +67,25 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
...
+
+Creating correlations has just become easier. Try out our new correlations editor in Explore by selecting the **+ Add > Add correlation** option from the top bar, or from the command palette. The editor shows all possible places where you can place data links and guides you through building and testing target queries. For more information, refer to [the documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/correlations/).
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:79]:
```suggestion
Creating correlations has just become easier. Try out our new correlations editor in **Explore** by selecting the **+ Add > Add correlation** option from the top bar or from the command palette. The editor shows all possible places where you can place data links and guides you through building and testing target queries. For more information, refer to [the documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/correlations/).
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 10]
```diff
@@ -67,3 +67,25 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
...
+
+To try out Correlations, enable the `correlations` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, contact Grafana Support.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:81]: Sorry still getting used to the separation of on-prem and Cloud release notes. Forgot this sentence doesn't make sense here. It'll be part of the Cloud version of these entries.
```suggestion
To try out Correlations, enable the `correlations` feature toggle.
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 11]
```diff
@@ -67,3 +67,25 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
...
+
+### Create correlations for provisioned data sources
+
+<!-- Piotr Jamróz -->
+
+_Available in public preview in Grafana Open Source, Enterprise and Cloud_
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:87]:
```suggestion
_Available in public preview in Grafana Open Source and Enterprise_
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 12]
```diff
@@ -67,3 +67,25 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
...
+
+You can now create correlations using either the **Administration** page or provisioning, regardless of whether a data source was provisioned or not. In previous versions of Grafana, if a data source was provisioned, the only way to add correlations to it was also with provisioning. Now, that's no longer the case, and you can easily create new correlations mixing both methods—using the **Administration** page or provisioning.
+
+To try out Correlations, enable the `correlations` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, contact Grafana Support.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:91]:
```suggestion
To try out Correlations, enable the `correlations` feature toggle.
```

**Review by imatwawana — COMMENTED** [review]
Apologies, I found a few small things that I missed on the first pass. I removed some of the Cloud related content as I've opened [a PR to add these items in the Cloud what's new](https://github.com/grafana/grafana/pull/76558) as well and I've tagged you for review.

File: docs/sources/whatsnew/whats-new-in-v10-2.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (imatwawana) [reply docs/sources/whatsnew/whats-new-in-v10-2.md]: I've addressed that in my review and opened a new PR per my review notes.

File: docs/sources/whatsnew/whats-new-in-v10-2.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (imatwawana) [reply docs/sources/whatsnew/whats-new-in-v10-2.md]: Yes, we definitely need to update it! It's on the list of tasks.

**Review by gelicia — APPROVED** [review]

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 13]
```diff
@@ -67,3 +67,25 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
...
+
+### Create correlations for provisioned data sources
+
+<!-- Piotr Jamróz -->
+
+_Available in public preview in Grafana Open Source and Enterprise_
+
+You can now create correlations using either the **Administration** page or provisioning, regardless of whether a data source was provisioned or not. In previous versions of Grafana, if a data source was provisioned, the only way to add correlations to it was also with provisioning. Now, that's no longer the case, and you can easily create new correlations mixing both methods—using the **Administration** page or provisioning.
+
+To try out Correlations, enable the `correlations` feature toggle.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:91]:
```suggestion
To try out **Correlations**, enable the `correlations` feature toggle.
```

**Review by imatwawana — APPROVED** [review]
Just one small formatting thing that I missed but otherwise this LGTM!

**Pull request merged**